depends/packages/librustzcash.mk

```diff
@@ -1,10 +1,14 @@
 package=librustzcash
 $(package)_version=0.1
-$(package)_download_path=https://github.com/zcash/$(package)/archive/
-$(package)_file_name=$(package)-$($(package)_git_commit).tar.gz
+#$(package)_download_path=https://github.com/zcash/$(package)/archive/
+#$(package)_file_name=$(package)-$($(package)_git_commit).tar.gz
+$(package)_download_path=https://github.com/ark2038/librustzcash/archive/
```

Comment:
instead of introducing another package on another repo, apply patch after download @ark2038

Reply:
Tracking dependencies in https://github.com/ZencashOfficial/zen/issues/295